server/src/db.d.ts

```diff
@@ -190,7 +190,7 @@ export interface Exif {
 }
 
 export interface FaceSearch {
-  embedding: string;
```

Comment:
I've just changed this for now since I believe that's just wrong. Let me know if I should fix it elsewhere